pkg/morph/event/netmap/update_peer_notary.go

```diff
@@ -0,0 +1,72 @@
+package netmap
+
+import (
+	"crypto/elliptic"
+	"fmt"
+
+	"github.com/nspcc-dev/neo-go/pkg/crypto/keys"
+	"github.com/nspcc-dev/neo-go/pkg/vm/opcode"
+	"github.com/nspcc-dev/neofs-api-go/pkg/netmap"
+	netmapv2 "github.com/nspcc-dev/neofs-api-go/v2/netmap"
+	"github.com/nspcc-dev/neofs-node/pkg/morph/event"
+)
+
+func (s *UpdatePeer) setPublicKey(v []byte) (err error) {
+	if v != nil {
```

Comment:
Is `v == nil` case correct?

Reply:
in fact, it is impossible to get nil slice from `neo-go` library here, but added check for it